ui/package.json

```diff
@@ -14,9 +14,7 @@
     "test:e2e": "exit 0"
   },
   "dependencies": {
-    "@agoric/react-components": "^0.1.1-dev-ca0ddde.0",
-    "@agoric/ui-components": "^0.3.9-u13.0",
```

Comment:
These 2 deps were just dead code, I suppose?

Reply:
Yea, ui-components wasn't imported anywhere, and web-components was only used to import a type that wasn't used anywhere